experimental/trusted_shuffler/scripts/run

```diff
@@ -2,12 +2,16 @@
 
 readonly SCRIPTS_DIR="$(dirname "$0")"
 
+readonly K=2
+readonly QPS=10
+readonly ROUNDS=2
```

Comment:
What do `ROUNDS` correspond to?
Does it mean, that if we have 1000 QPS, then 2 rounds will mean 2 milliseconds?
Could we have `SECONDS` instead?

Reply:
I think the fact that 1 round == 1 second should be explained / configured somewhere, it does not follow from the fact that we are using QPS for measuring the "speed".

Reply:
For instance, I think the client should be configurable with either a timeout after which it stops (e.g. "run at 10QPS for 20 seconds", or just run forever until interrupted (e.g. "run at 10QPS until I tell you to stop").